common/firmware/spi/spi.c

```diff
@@ -117,12 +117,20 @@ void SPI2_init() {
     handle = MX_SPI2_Init();
 }
 
+void get_SPI_handle(void* tmp){
+    SPI_HandleTypeDef handle = MX_SPI2_Init();
```

Comment:
Can this function return a `SPI_HandleTypeDef *` that is globally allocated in this file?

Reply:
[2e47656](https://github.com/Opentrons/ot3-firmware/pull/145/commits/2e476562682799348290143c475adbb72ead3dc9) makes spi.c and spi_comms.cpp less complicated.